assets/src/dashboard/utils/useTemplateView.js

```diff
@@ -32,10 +33,12 @@ import { PageSizePropType } from '../types';
 import { clamp, usePagePreviewSize } from './index';
 
 export default function useTemplateView({ totalPages }) {
+  const enableTemplatePreviews = useFeature('enableTemplatePreviews');
+
   const [searchKeyword, _setSearchKeyword] = useState('');
   const [sort, _setSort] = useState(TEMPLATES_GALLERY_SORT_OPTIONS.POPULAR);
   const [page, setPage] = useState(1);
-  const [previewVisible, setPreviewVisible] = useState();
+  const [previewVisible, _setPreviewVisible] = useState();
```

Comment:
Is the `_` prefix a new practice we've adopted for setters in the dashboard?

Reply:
ohh I see. it's just if it's internal to the actual setter when you wanna add some logic to setting something. cewl